bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/client/SerialClient.java

```diff
@@ -36,6 +36,8 @@
 @NonNullByDefault
 public class SerialClient extends AbstractKNXClient {
 
+    private final String CALIMERO_ERROR_CANNOT_OPEN_PORT = "failed to open serial port";
```

Comment:
Sorry but `private static final` is expected.

Reply:
done